examples/kotlin/BUILD.bazel

```diff
@@ -11,8 +11,8 @@ load("@npm//http-server:index.bzl", "http_server")
 # Grab a Maven dependency
 kt_js_import(
     name = "kotlinx-html-js",
-    jars = ["@maven//:v1/http/dl.bintray.com/kotlin/kotlinx.html/org/jetbrains/kotlinx/kotlinx-html-js/0.6.12/kotlinx-html-js-0.6.12.jar"],
-    srcjar = "@maven//:v1/http/dl.bintray.com/kotlin/kotlinx.html/org/jetbrains/kotlinx/kotlinx-html-js/0.6.12/kotlinx-html-js-0.6.12-sources.jar",
+    jars = ["@maven//:v1/https/repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-html-js/0.7.3/kotlinx-html-js-0.7.3.jar"],
```

Comment:
Does 0.6.13 work? https://dl.bintray.com/kotlin/kotlin-dev/org/jetbrains/kotlinx/kotlinx-html-js/0.6.13-build-1696/kotlinx-html-js-0.6.13-build-1696.jar

Reply:
I didn't realize that any bintray URLs still worked... the registry doesn't respond so you can't do the `bazel run @unpinned_maven//:pin` with any bintray urls